### PR TITLE
#167190456 Authorized Users can report an article

### PR DESCRIPTION
**Pull request by chiboycalix**
[Finishes #167190456]
#### What are the relevant pivotal tracker stories?
[#167190456](https://www.pivotaltracker.com/story/show/167190456)
#### Screenshots (if appropriate)

Screenshot 2019-08-07 at 11 38 13 AM

### CODE REVIEW

File: server/controllers/reportController.js [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+  /**
+   * @description Creates an Article
+   * @param {Object} req - request object
+   * @param {Object} res - response object
+   * @returns {Object} Article data
+   * @memberof ReportController
+   */
+  static async getOneReport(req, res) {
+    const a = req.params.reportId;
+    const r = /^[0-9]*$/.test(a);
```
Comment (chuxmykel) [inline server/controllers/reportController.js]: Is there a reason why you're doing your param validation in the controller?

File: server/tests/report.test.js [hunk 2]
```diff
@@ -0,0 +1,158 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import dotenv from 'dotenv';
+import app from '../index';
+import reportData from './testData/report.data';
+
+
+dotenv.config();
```
Comment (chuxmykel) [inline server/tests/report.test.js]: You imported dotenv mistakenly bro. Check it out.

**Review by chuxmykel — APPROVED** [review]
Address the issues and then you're good to go.

File: server/tests/report.test.js (hunk 2 above)
Comment (chiboycalix) [reply server/tests/report.test.js]: Okay. Thanks for pointing this out. I will fix it

File: server/controllers/reportController.js (hunk 1 above)
Comment (chiboycalix) [reply server/controllers/reportController.js]: It's a small check, and it would be an overkill to create a whole new file just for it

Comment (chuxmykel) [reply server/controllers/reportController.js]: I don't think you need a whole new file. You can just write a schema for it. What do you think?

**Review by Hadeneekeh — APPROVED** [review]
Aside from raised concerns, everything looks good to me!

File: server/controllers/reportController.js (hunk 1 above)
Comment (chiboycalix) [reply server/controllers/reportController.js]: oh. nice. I'll do that

**Review by Lundii — APPROVED** [review]
LGTM

**Review by topseySuave — APPROVED** [review]

**Pull request closed without merging**